src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -194,7 +194,16 @@ public static function register($mode = 0)
     private static function hasColorSupport()
     {
         if ('\\' === DIRECTORY_SEPARATOR) {
-            return false !== getenv('ANSICON') || 'ON' === getenv('ConEmuANSI') || 'xterm' === getenv('TERM');
+            return
+                defined('PHP_WINDOWS_VERSION_MAJOR')
```

Comment:
I think we can safely state that only Windows does `'\\' === DIRECTORY_SEPARATOR`